spi_nand_flash/src/dhara_glue.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * SPDX-FileCopyrightText: 2022 mikkeldamsgaard project
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * SPDX-FileContributor: 2015-2023 Espressif Systems (Shanghai) CO LTD
+ */
+
+#include "dhara/nand.h"
+#include "esp_check.h"
+#include "esp_err.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "spi_nand_oper.h"
+#include "spi_nand_flash.h"
+#include "nand.h"
+
+#define ROM_WAIT_THRESHOLD_US 1000
+
+static const char *TAG = "dhara_glue";
+
+esp_err_t wait_for_ready(spi_device_handle_t device, uint32_t expected_operation_time_us, uint8_t *status_out)
```

Comment:
Seems like it is used only from this source file, so it could be `static`?

Reply:
It is used by othe file as well e.g. src/nand.c